roles/kubernetes/control-plane/tasks/main.yml

```diff
@@ -105,6 +105,13 @@
 - name: Include kubeadm secondary server apiserver fixes
   include_tasks: kubeadm-fix-apiserver.yml
 
+- name: Cleanup unused AuthorizationConfiguration file versions
+  file:
+    path: "{{ kube_config_dir }}/apiserver-authorization-config-{{ item }}.yaml"
+    state: absent
+  loop: "{{ '[\"v1alpha1\", \"v1beta1\", \"v1\"]' | from_json | reject('equalto', kube_apiserver_authorization_config_api_version) | list }}"
```

Comment:
I think you don't need from_json, can't you specify the array directly in Jinja ?

Reply:
I tried that, but ran into issues where `reject` interpreted the string as a character array and I got `from_json` working more quickly to convert from string to object.

I can try it again with: 
```yaml
  loop: "{{ ['v1alpha1', 'v1beta1', 'v1'] | reject('equalto', kube_apiserver_authorization_config_api_version) | list }}"
```

Which should work.